crates/cheatnet/src/runtime_extensions/forge_runtime_extension/mod.rs

```diff
@@ -868,8 +870,15 @@ pub fn get_all_used_resources(runtime: ForgeRuntime) -> UsedResources {
 
     let execution_resources = top_call.borrow().used_execution_resources.clone();
     let top_call_syscalls = top_call.borrow().used_syscalls.clone();
+    let events = runtime_call_info
+        .execution
+        .events
+        .iter()
+        .map(|ordered_event| ordered_event.event.clone())
+        .collect();
```

Comment:
Is there no method for that? It seems like we are ignoring events from inner calls (unless events are like resources not like messages in the sense that they include the events from inner calls - then the name `top_call_events` above is missleading)

Reply:
My mistake, i added a test for this. Was missing the inner events, which are now collected